### PR TITLE
open-cluster-management/backlog#1352623 Add Pwr z pol note

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: risk_compliance/image_vuln_policy.adoc [hunk 1]
```diff
@@ -6,7 +6,11 @@ Apply the image vulnerability policy to detect if container images have vulnerab
...
 For more information about the Security Operator, see the _Container Security Operator_ from the https://github.com/quay/container-security-operator[Quay repository].
 
-*Note:* Image vulnerability policy is not functional during a disconnected installation.
+*Notes:*
+
+* Image vulnerability policy is not functional during a disconnected installation.
+
+* The link:https://github.com/open-cluster-management/grc-ui/blob/main/src-web/components/common/templates/spec-imagemanifestvuln.yaml[Image vulnerability policy] is not supported on the IBM Power and IBM Z architectures. It relies on the link:https://operatorhub.io/operator/project-quay-container-security-operator[Quay Container Security Operator]. There are no `ppc64le` or `s390x` images in the link:https://quay.io/repository/quay/container-security-operator[container-security-operator registry], and `GOARCH=amd64` is hardcoded in the link:https://github.com/quay/container-security-operator/blob/master/Dockerfile#L18[Dockerfile] that is provided in the link:https://github.com/quay/container-security-operator[container-security-operator GitHub repository].
```
Comment (jaywcarman) [inline risk_compliance/image_vuln_policy.adoc]: I think the note about `GOARCH=amd64` in the Dockerfile might be too much technical detail here - the important piece that is that no `ppc64le` or `s390x` images are provided in the registry.

Comment (cadawson) [reply risk_compliance/image_vuln_policy.adoc]: Thanks. I removed that piece. I thought that might be a clue to a workaround.

**Review by jaywcarman — APPROVED** [review]
LGTM

**Comment by cadawson** [comment]
@dockerymick Heads-up that this was an update for the Power and z limitations. Let me know if you have concerns. I clicked the merge button too early. Thanks!

**Comment by dockerymick** [comment]
@cadawson no worries, I'll take a look soon

File: risk_compliance/image_vuln_policy.adoc [hunk 2]
```diff
@@ -6,7 +6,11 @@ Apply the image vulnerability policy to detect if container images have vulnerab
 The image vulnerability policy is checked by the Kubernetes configuration policy controller.
 For more information about the Security Operator, see the _Container Security Operator_ from the https://github.com/quay/container-security-operator[Quay repository].
 
-*Note:* Image vulnerability policy is not functional during a disconnected installation.
+*Notes:*
+
+* Image vulnerability policy is not functional during a disconnected installation.
+
+* The link:https://github.com/open-cluster-management/grc-ui/blob/main/src-web/components/common/templates/spec-imagemanifestvuln.yaml[Image vulnerability policy] is not supported on the IBM Power and IBM Z architectures. It relies on the link:https://operatorhub.io/operator/project-quay-container-security-operator[Quay Container Security Operator]. There are no `ppc64le` or `s390x` images in the link:https://quay.io/repository/quay/container-security-operator[container-security-operator registry].
```
Comment (dockerymick) [inline risk_compliance/image_vuln_policy.adoc:13]: @cadawson I'm not sure that the link to the Image vulnerability yaml is the correct link. I believe this is the correct link: https://github.com/open-cluster-management/policy-collection/blob/main/stable/SI-System-and-Information-Integrity/policy-imagemanifestvuln.yaml

**Pull request merged**